### PR TITLE
Account form classes

### PR DESCRIPTION
**Pull request by PeteyRev**
#### What?

When editing Account Signup Fields in the Control Panel, there are fields for 
`CSS Class Name, Inline CSS Style, Field Size` that can be customized by the user, however these do not currently work. The data is present , it just was never added via handlebars

#### Screenshots (if appropriate)

Attach images or add image links here.

oBundle - Lead Dev Sandbox - BigCommerce Control Panel
oBundle - Lead Dev Sandbox - Create Account

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "bigcommerce-cornerstone",
-  "version": "4.4.1",
+  "version": "4.5.0",
```
Comment (junedkazi) [inline package-lock.json:3]: @PeteyRev thank you for your contribution. Can you please rebase and drop the `package-lock.json` change. We do need to fix it and we are going to fix our release script to make sure this does not happen again.

Comment (PeteyRev) [reply package-lock.json]: @junedkazi Should be gtg

File: package-lock.json [hunk 2]
```diff
@@ -9846,6 +9860,7 @@
             "hoek": "2.x.x"
           }
         },
+<<<<<<< HEAD
```
Comment (junedkazi) [inline package-lock.json:9863]: @PeteyRev I don't believe the rebase went well since I am seeing these line.

File: package-lock.json [hunk 3]
```diff
@@ -9862,6 +9877,12 @@
           "version": "6.0.0",
           "resolved": "https://registry.npmjs.org/wreck/-/wreck-6.0.0.tgz",
           "integrity": "sha1-T0CGaWHl14rOBPMqa38x8/PFFjg=",
+=======
```
Comment (junedkazi) [inline package-lock.json:9880]: same here

File: package-lock.json [hunk 4]
```diff
@@ -9862,6 +9877,12 @@
...
+        "wreck": {
+          "version": "6.3.0",
+          "resolved": "https://registry.npmjs.org/wreck/-/wreck-6.3.0.tgz",
+          "integrity": "sha1-oTaXafB7u2LWo3gzanhx/Hc8dAs=",
+>>>>>>> 4bd50865cf4e39814a9a0c485c5643cc99799020
```
Comment (junedkazi) [inline package-lock.json:9885]: same here

**Pull request merged**